package.json

```diff
@@ -3,6 +3,13 @@
 	"version": "0.0.3",
 	"description": "Color space agnostic color manipulation library",
 	"main": "color.js",
+	"exports": {
+		"import": "./color.js",
```

Comment:
Please use `"./dist/color.esm.js:` here, `color.js` on the root is primarily for use by the color.js website, and assigns Color to a global (so that people can play in the console) which kind of defeats the purpose of modules if used generally.